package.json

```diff
@@ -3,17 +3,20 @@
   "version": "2.7.12",
   "description": "Lightning Renderer for Solid Universal",
   "type": "module",
-  "source": "src/index.ts",
```

Comment:
what is source field for btw? The only other mention of it that I could find was here: http://parcel2-docs.vercel.app/configuration/package-json/#source

Reply:
I'm not sure... 🙃